### DISCUSSION TITLE
WalletConnect connectivity issue

### DISCUSSION THREAD

**Discussion by shreym0** [WalletConnect/walletconnect-monorepo#5281] [Developer Support]
I am using the walletconnect wagmi in my application and facing automatic disconnect from the walletconnect on refresh of page.
I am using the following versions

    "@wagmi/core": "^2.6.17",
    "@web3modal/wagmi": "^4.1.9",
        "viem": "^2.9.23",
        
Is it due to the older version or any other reason ?

**Comment by rtomas** [comment]
Hi @shreym0, I'm not sure. It really could be an old bug.
It would be great to update to latest.

**Comment by Ratheeshriyan** [comment]
iam using php it is possible to connect
